Sources/PT.PM.AntlrUtils/AntlrUtils.cs

```diff
@@ -16,19 +17,52 @@ namespace PT.PM.AntlrUtils
 {
     public static class AntlrUtils
     {
-        public static string GetText(this ParserRuleContext ruleContext, IList<IToken> tokens)
+        private static long processedFilesCount;
+        private static long processedBytesCount;
+        private static long checkNumber;
+        private static volatile bool excessMemory;
+
+        public static long MemoryConsumptionBytes { get; set; } = 2 * 1024 * 1024 * 1024L;
+
+        public static long ClearCacheFilesBytes { get; set; } = 5 * 1024 * 1024L;
+
+        public static int ClearCacheFilesCount { get; set; } = 50;
```

Comment:
Three static fields above are named a bit unclear. Are they limits? Or increments? I suggest that the names should reflect that.
For instance, `MaxAllocatedMemoryBytes` or `MemoryLimitInBytes`; `MaxCacheSizeInBites`; `MaxFilesInCache` or `MaxCachedFiles`.